### PR TITLE
Preparations for experimental mutator integration

### PR DESCRIPTION
**Pull request by fmeum**

### CODE REVIEW

File: src/main/java/com/code_intelligence/jazzer/mutation/support/InputStreamSupport.java [hunk 1]
```diff
@@ -51,8 +51,11 @@ public static byte[] readAllBytes(InputStream stream) throws IOException {
     byte[] result = new byte[arrayLength];
     int offset = 0;
     byte[] buffer;
+    int remaining = arrayLength;
```
Comment (bertschneider) [inline src/main/java/com/code_intelligence/jazzer/mutation/support/InputStreamSupport.java:54]: Why is this necessary? I expected sum of `buffer.length` to be `arrayLength`.

**Review by bertschneider — APPROVED** [review]
Nice cleanup!

File: src/main/java/com/code_intelligence/jazzer/mutation/support/InputStreamSupport.java (hunk 1 above)
Comment (fmeum) [reply src/main/java/com/code_intelligence/jazzer/mutation/support/InputStreamSupport.java:54]: That's not necessarily true for the last buffer, which is exactly what caused the bug. I feel a bit better now about missing this at first :-D
